tests/acceptance/helpers/filesFoldersHelpers.js

```diff
@@ -0,0 +1,50 @@
+const { client } = require('nightwatch-api')
```

Comment:
`filesFoldersHelpers.js` should be `filesFoldersHelper.js`

Reply:
fixed